src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -194,6 +194,10 @@ internal Canvas expansionBay
         private static BoolToVisibilityConverter _booleanToVisibilityConverter = new BoolToVisibilityConverter();
         private static EmptyToVisibilityCollapsedConverter _emptyToVisibilityCollapsedConverter = new EmptyToVisibilityCollapsedConverter();
         private static ZoomToVisibilityCollapsedConverter _zoomToVisibilityCollapsedConverter = new ZoomToVisibilityCollapsedConverter();
+        private static IValueConverter _sZoomFadeControl = SharedDictionaryManager.DynamoModernDictionary["SZoomFadeControl"] as IValueConverter;
```

Comment:
I don't believe having the `_` to name private members aligns with the original coding standards for Dynamo. I'm not sure when this new style started making its way into the code.